webapp/src/components/token-usage/TokenUsageBar.tsx

```diff
@@ -0,0 +1,38 @@
+import { Popover, PopoverSurface, PopoverTrigger } from '@fluentui/react-components';
+import { Constants } from '../../Constants';
+
+interface ITokenUsageBar {
+    functionName: string;
+    functionUsage: number;
+    totalUsage: number;
+    color: string;
+}
+
+export const TokenUsageBar: React.FC<ITokenUsageBar> = ({ functionName, functionUsage, totalUsage, color }) => {
+    const percentage = functionUsage / totalUsage;
+    const barWidth = percentage * Constants.MAX_BAR_GRAPH_WIDTH;
```

Comment:
I don't think we need an app-level constant for this, seems more appropriate as a file-level constant. especially considreing that the max width will depend on how/where the bar is rendered?

Reply:
Good catch, I was sharing it with another component but reverted the changes. Removed